### PR TITLE
French localization

### PR DESCRIPTION
**Pull request by Risengan**
Thanks for your work! 😉

### CODE REVIEW

**Comment by MhdHejazi** [comment]
Thank you for your contribution!

Actually, only `Localizable.strings` file should be translated, the strings in the storyboards are just placeholders. Sorry for not making this clear enough.

Could you please remove the other files?

**Comment by Risengan** [comment]
Is it good? I'm not a Git expert 😅

**Comment by MhdHejazi** [comment]
It's great. Thanks!

File: Corona/Localization/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+"data.updating" = "Mise à jour...";
+"data.updateDate" = "Dernière mise à jour :";
+"data.errorTitle" = "Impossible de mettre à jour les données";
+"data.errorMessage" = "Veuillez vous assurer que vous êtes connecté à Internet.";
+"app.newVersionTitle" = "Nouvelle version disponible";
+"app.newVersionMessage" = "Veuillez mettre à jour à partir de %@";
+"message.cancel" = "Annuler";
+"message.open" = "Ouvrir";
+"message.ok" = "OK";
+"message.done" = "OK";
```
Comment (MhdHejazi) [inline Corona/Localization/fr.lproj/Localizable.strings]: I guess "Done" should not be the same as "OK".

This is the context:

image

To see it: Tap the (⠸) button, then "Share"

Comment (Risengan) [reply Corona/Localization/fr.lproj/Localizable.strings]: It’s the best choice in French 😉

Comment (MhdHejazi) [reply Corona/Localization/fr.lproj/Localizable.strings]: Yeah, it looks like Apple is also using OK for both OK & Done. Thanks!

**Pull request merged**